### ISSUE TITLE
What is the current state of Post 2 Post for large projects?

### ISSUE THREAD

**Issue by cggit** [scribu/wp-posts-to-posts#501]
I love this plugin but am not currently comfortable depending on it for key features in large projects.
Scribu stated publicly years ago that he will no longer be maintaining the plugin but I see some activity here and there which makes me excited to maybe use it again especially since I've yet to find anything that compares. 

My question is, should we be moving away from the this plugin or has Scribu or someone else made any commitment to making sure it at least doesn't break as WP updates?

**Comment by scribu** [comment]
The minor updates you saw were all done by contributors (I just pushed them to wordpress.org), so there are at least people who continue to use it.

**Comment by cggit** [comment]
Hi Scribu, but do you have any vested interest in seeing the plugin live on? I get that you've stopped support but is maintenance completely done by contributors or would you fix something a WP update broke if no one else did? WordPress misses you!

**Comment by scribu** [comment]
> do you have any vested interest in seeing the plugin live on?

No, not really. I would like it if someone found it useful enough to take over maintenance of it, but I'm not going to lose any sleep if that doesn't happen.
